### ISSUE TITLE
Problem with search boxes for name/address on main page-across all installations

### ISSUE THREAD

**Issue by EveSnyder** [historyforge/historyforge#249]
I've gotten a few messages about the search boxes on the main page not working recently-it appears you have to enter three characters into the boxes before any suggested records appear and that for name it has to be the first name. Some people are giving up when nothing appears especially when they type in a last name that should be there and nothing comes up. It looks like the search is broken.

**Comment by PatientRowan** [comment]
Some examples:
Old Ebenezer Miller who built my house and appears in censuses 1910, 1920, 1930, does not currently come up in the public search box under either Ebenezer or Miller.
A lady who works at my haircutter’s is named Kenjerska, which I recognized as a Problem Family Name from the census.  Was I ever excited!  I tried to show her her family on HistoryForge and ask some more about them.  Kenjerska from the public page doesn’t retrieve anything.  I got them through the All People index but as a demo it was a bit of an anticlimax.
